variables.tf

```diff
@@ -136,12 +136,27 @@ variable "enable_ipv6" {
   EOF
 }
 
+variable "enable_ipv6_only" {
+  type = bool
+  default = false
+}
+
 variable "enable_kube_span" {
   type        = bool
   default     = false
   description = "If true, the KubeSpan Feature (with \"Kubernetes registry\" mode) will be enabled."
 }
 
+variable "ipv4_nameservers" {
+  type = list(string)
+  default = ["4.4.4.4", "8.8.8.8"]
+}
+
+variable "ipv6_nameservers" {
+  type = list(string)
+  default = ["2a00:1098:2b::1", "2a00:1098:2c::1", "2a01:4f8:c2c:123f::1"] # https://nat64.net/ DNS64 Servers
```

Comment:
I've used this servers just to make it work ... but in truth someone that want's to have a ipv6 only setup would need something like this.

Need to investigate: does the corends inside the cluster also needs an IPV6 forward config?